nuxt.config.js

```diff
@@ -40,10 +40,10 @@ export default {
 		meta: [
 			{ charset: "utf-8" },
 			{ name: "viewport", content: "width=device-width, initial-scale=1" },
-			{ hid: "description", name: "description", content: process.env.npm_package_description || "" }
+			{ hid: "og:image", property: "og:image", content: "/og-image.png" } // TODO: Customize per page.
```

Comment:
It seems the `og:url` meta data is missing. According to [The Open Graph Protocol](https://ogp.me) this is a required property.